wordpress-mobile/WordPress-iOS
# Share Extension: Increase bottom padding in editor

Fixes #8591.

Stalebot just closed #8951, but I figured it looked simple enough to fix so I've taken a stab at it.

![share-bottom-padding](https://user-images.githubusercontent.com/4780/52569195-80617f00-2e08-11e9-8b71-969b29016f2c.png)

**To test:**

* Build and run
* Go to another app (e.g. Safari) and launch the Share extension.
* Type some long content that causes the editor in the extension to scroll.
* Verify that there's some padding at the bottom of the editor (see screenshot above)

Update release notes:
